x-pack/plugins/infra/public/pages/logs/log_entry_rate/page_results_content.tsx

```diff
@@ -232,25 +220,23 @@ export const LogEntryRateResultsContent: React.FunctionComponent<LogEntryRateRes
             onRecreateMlJobForUpdate={viewSetupFlyoutForUpdate}
           />
         </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiPanel paddingSize="m">
-            <LogRateResults
-              isLoading={isLoading}
-              results={logEntryRate}
-              setTimeRange={handleChartTimeRangeChange}
-              timeRange={queryTimeRange.value}
-            />
-          </EuiPanel>
-        </EuiFlexItem>
         <EuiFlexItem grow={false}>
           <EuiPanel paddingSize="m">
             <AnomaliesResults
-              isLoading={isLoading}
+              isLoading={isLoading || isLoadingLogEntryAnomalies}
               viewSetupForReconfiguration={viewSetupFlyoutForReconfiguration}
               results={logEntryRate}
+              anomalies={logEntryAnomalies}
               setTimeRange={handleChartTimeRangeChange}
               timeRange={queryTimeRange.value}
               jobId={jobIds['log-entry-rate']}
```

Comment:
This is currently passed to the expanded rows regardless and will be incorrect for category anomaly rows. Could it be removed here and extracted from the anomaly record locally inside of each row?

Reply:
Sorry, I don't follow?

Reply:
Hm, GitHub didn't highlight the line I was referring to :thinking: I was talking about the `jobId` here, which needs to be different for rows depending on the kind of anomaly. It might therefore be reasonable to extract it from the anomaly record in each row instead of passing just one of the ids in this level.

Reply:
Ah, I see. Thanks for clarifying.